manifests/charts/istio-control/istio-discovery/files/waypoint.yaml

```diff
@@ -84,7 +84,7 @@ spec:
       tolerations:
       {{- toYaml .Values.global.waypoint.tolerations | nindent 8 }}
       {{- end }}
-      terminationGracePeriodSeconds: 2
```

Comment:
I think you should add this to the values.yaml file to get rid of the `| default 2`